Assignment4/Tests.java

```diff
@@ -0,0 +1,40 @@
+package Assignment4;
+
+import org.junit.Test;
+
+import junit.framework.*;
+
+public class Tests {
+

```

Comment:
Test classes are often easier to read if you have lots of small tests testing particular conditions, and give then good names (instead of test1, test2). So for example:

testIslandInTopCorner()
testIslandOnWestEdge()
testDiagonalIsntConnection()
testUShapedIsland()

That way if you or someone else changes your code, you know exactly what it breaks for.